src/components/Tool/Appeal/List/AppealTour/AppealTour.tsx

```diff
@@ -0,0 +1,233 @@
+import React, { useEffect } from 'react';
+import styled from '@emotion/styled';
+import { Alert, AlertTitle, Box, Button, Typography } from '@mui/material';
+import { TFunction, useTranslation } from 'react-i18next';
+import theme from 'src/theme';
+import {
+  AppealTourEnum,
+  AppealsContext,
+  AppealsType,
+} from '../../AppealsContext/AppealsContext';
+
+const StatisticsSummary = styled(Box)(() => ({
+  display: 'flex',
+  flexWrap: 'nowrap',
+  marginTop: theme.spacing(1),
+  marginBottom: theme.spacing(1),
+  '@media (max-width: 550px)': {
+    flexWrap: 'wrap',
+  },
+}));
+
+const StatisticBox = styled(Box, {
+  shouldForwardProp: (prop) => prop !== 'color',
+})<{ color: string }>(({ color }) => ({
+  backgroundColor: color,
+  textAlign: 'center',
+  padding: `${theme.spacing(2)} ${theme.spacing(2)}`,
+  position: 'relative',
+  width: '20%',
+  '&:after': {
+    content: '" "',
+    width: '24px',
+    height: '24px',
+    position: 'absolute',
+    right: '-12px',
+    zIndex: '50',
+    background: color,
+    top: '50%',
+    transform: 'translateY(-50%) rotate(45deg)',
+  },
+  '@media (max-width: 550px)': {
+    width: '100%',
+    padding: `${theme.spacing(1)} ${theme.spacing(1)}`,
+    '&:after': {
+      width: '16px',
+      height: '16px',
+      top: '100%',
+      right: '50%',
+      transform: 'translate(50%, -50%) rotate(45deg)',
+    },
+  },
+}));
+
+type ActionButtonsProps = {
+  nextTourStep: () => void;
+  hideTour: () => void;
+  tourStep: AppealTourEnum;
+  t: TFunction;
+};
+const ActionButtons = ({
+  nextTourStep,
+  hideTour,
+  tourStep,
+  t,
```

Comment:
Is there a reason `t` is a prop instead of using `useTranslation` inside the component?

Reply:
I think I thought it would be less work for React to do. I don't have a preference so I'll add `useTranslation` inside the component.